docs/source/concepts/glossary.rst

```diff
@@ -7,9 +7,9 @@ Glossary of concepts
 DAML
 ****
 
-**DAML** is a programming language for writing `smart contracts <#contract-contract-instance>`__, that you can use to build an application based on a `ledger <#ledger-daml-ledger>`__. You can run DAML contracts on many different ledgers.
+**DAML** is a programming language for writing `smart contracts <#contract>`__, that you can use to build an application based on a `ledger <#ledger-daml-ledger>`__. You can run DAML contracts on many different ledgers.
 
-Contract, contract instance
```

Comment:
This is why I needed to change every ref from `contract-contract-instance` to `contract`